internal/schema/schemas/posit-publishing-schema-v3.json

```diff
@@ -350,6 +350,7 @@
               "python-dash",
               "python-fastapi",
               "python-flask",
+              "python-gradio",
```

Comment:
I noticed that `python-gradio` wasn't listed here to enforce that the `python` section was required if the `type = 'python-gradio'`.